handbook/business-operations/README.md

```diff
@@ -858,6 +858,35 @@ To make a request of the business operations department, [create an issue using
 
 > If you're not sure that your request can wait that long, then please ask for urgent help in our group Slack channel: `#g-business-operations`.  Only use this approach or at-mention contributors in business operations directly in urgent situations.  Otherwise, create an issue.
 
+## Contract review
+
+> If a document is ready for signature and does not need to be reviewed or negotiated, you can skip the review process and use the signature process documented further down in this handbook section.
+
+**To get a contract reviewed:**
```

Comment:
Would recommend using a header (`####`) instead of bold text. (idk what the norm is @ Fleet, but in markdown I prefer to use headers for stuff that's on its own line and bold for stuff that's being emphasized inline in a block of text.)

```suggestion
#### To get a contract reviewed:
```